app/controllers/insured/product_shoppings_controller.rb

```diff
@@ -8,6 +8,7 @@ class ProductShoppingsController < ApplicationController
 
     # rubocop:disable Metrics/CyclomaticComplexity
     def continuous_show
+      authorize @hbx_enrollment
```

Comment:
What do you think about moving it to `before_action` for all those actions?

Reply:
we're not doing it for the following reasons:

i) It will appear in reports again saying missing authorization.
ii) The patterns we established as part of NuHarbor fixes is to have one policy definition for each action and adding this in our controller methods before doing further processing (also, having this explicitly defined in each action makes it easier to read)